crates/pathfinder/src/storage.rs

```diff
@@ -160,15 +178,19 @@ mod tests {
         assert_eq!(version, DB_VERSION_EMPTY);
     }
 
-    #[test]
-    fn full_migration() {
-        let mut conn = rusqlite::Connection::open_in_memory().unwrap();
-        let transaction = conn.transaction().unwrap();
+    mod full_migration {
```

Comment:
I think just keeping to a single `full_migration` test case would be sufficient for forever -- the only additional thing is testing the actual schema result. But that can wait.

Reply:
Ah yeah, that's collateral from previous code removal :sweat_smile: